mypy/test/testparse.py

```diff
@@ -1,6 +1,5 @@
 """Tests for the mypy parser
-
-Test case descriptions are in files test/data/parse[-errors].test."""
+"""
```

Comment:
Proper docstring style is actually to have the close """ on the same line *iff* it's a single line. So it was wrong, and it's still wrong. :-)